base_geoengine_demo/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,20 @@
+* Nicolas Bessi <nicolas.bessi@camptocamp.com>
+* Frederic Junod <frederic.junod@camptocamp.com>
+* Yannick Vaucher <yannick.vaucher@camptocamp.com>
+* Sandy Carter <sandy.carter@savoirfairelinux.com>
+* Laurent Mignon <laurent.mignon@acsone.eu>
+* Jonathan Nemry <jonathan.nemry@acsone.eu>
+* David Lasley <dave@dlasley.net>
+* Daniel Reis <dgreis@sapo.pt>
+* Matthieu Dietrich <matthieu.dietrich@camptocamp.com>
+* Alan Ramos <alan.ramos@jarsa.com.mx>
+* Damien Crier <damien.crier@camptocamp.com>
+* Cyril Gaudin <cyril.gaudin@camptocamp.com>
+* Pierre Verkest <pverkest@anybox.fr>
+* Benjamin Willig <benjamin.willig@acsone.eu>
+* Devendra Kavthekar <dkatodoo@gmail.com>
+* Emanuel Cino <ecino@compassion.ch>
+* Thomas Nowicki <thomas.nowicki@camptocamp.com>
+* Alexandre Saunier <alexandre.saunier@camptocamp.com>
+* Sandip Mangukiya <smangukiya@opensourceintegrators.com>
+* Daniel Reis <dreis@opensourceintegrators.com>
```

Comment:
This list is probably not accurate as not every one that contributed to base_geoengine_demo contributed to base_geoengine. And you are twice there :)

Reply:
Fair enough. Did my best approach at that.